templates/etc/td-agent/td-agent.conf

```diff
@@ -38,7 +38,8 @@
   buffer_type file
   buffer_path /var/log/google-fluentd/buffers
   # Set the chunk limit conservatively to avoid exceeding the recommended
-  # chunk size of 5MB per write request.
```

Comment:
Do we need the same change in [windows-installer/fluent-template.conf](https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/windows-installer/fluent-template.conf#L33-L34)?

Reply:
Updated.